ce/ce/cli/markdown-table.ts

```diff
@@ -2,6 +2,36 @@
 // Licensed under the MIT License.
 
 import { strict } from 'assert';
+import { blue, gray, green, white } from 'chalk';
+import * as renderer from 'marked-terminal';
+
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const marked = require('marked');
+
+// setup markdown renderer
+marked.setOptions({
+  renderer: new renderer({
+    tab: 2,
+    emoji: true,
+    showSectionPrefix: false,
+    firstHeading: green.underline.bold,
+    heading: green.underline,
+    codespan: white.bold,
+    link: blue.bold,
+    href: blue.bold.underline,
+    code: gray,
+    tableOptions: {
+      chars: {
+        'top': '', 'top-mid': '', 'top-left': '', 'top-right': ''
+        , 'bottom': '', 'bottom-mid': '', 'bottom-left': '', 'bottom-right': ''
+        , 'left': '', 'left-mid': '', 'mid': '', 'mid-mid': ''
+        , 'right': '', 'right-mid': '', 'middle': ''
+      }
+    }
+  }),
+  gfm: true,
+});
```

Comment:
what's up with all these options?

Reply:
It's using a markdown renderer to generate tables. This sets the renderer options to create pretty tables.